packages/blocks/src/api/raw-handling/head-remover/index.js

```diff
@@ -0,0 +1,12 @@
+export default function( node ) {
```

Comment:
Just noting that, unlike the other filters, this has its own folder. Any reason why?